docs/firefox-accounts.rst

```diff
@@ -18,9 +18,10 @@ a non-production environment requires some additional Firefox profile configurat
 Local Development
 -----------------
 
-#. Set ``FXA_IFRAME_SRC`` in your ``.env`` file to point to the FXA development server:
+#. Add the following config settings to your ``.env`` file:
 
     ``FXA_IFRAME_SRC=https://stomlinson.dev.lcip.org/``
+    ``CSP_EXTRA_FRAME_SRC=https://stomlinson.dev.lcip.org/``
```

Comment:
Should this and `FXA_IFRAME_SRC above actually have single quotes around the URLs?